src/components/ElevatedButton/index.tsx

```diff
@@ -5,14 +5,24 @@ import classNames from 'classnames';
 interface ElevatedButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   children: React.ReactNode;
   isFullWidth: boolean;
+  className?: string;
 }
 
-const ElevatedButton = ({ children, isFullWidth, ...rest }: ElevatedButtonProps): JSX.Element => {
+const ElevatedButton = ({
+  children,
+  isFullWidth,
+  className = 'bg-white text-black-chinese',
+  ...attributes
```

Comment:
```suggestion
  ...rest
```

Reply:
@nvminhtue thanks, fixed in [7716683](https://github.com/manh-t/react-survey/pull/88/commits/7716683de5b6720fb6059e948fd25a6f2c2bcb8e)